gateway/gateway-requests/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 # Copyright 2020 - Nym Technologies SA <contact@nymtech.net>
-# SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
you don't want to be changing this one, it's part of client library to interact with the gateway (and not the actual gateway). this comment applies to everything in `gateway-requests`

Reply:
Done